shared/utils/combine-remediations.py

```diff
@@ -446,23 +447,23 @@ def main():
 
                             fix.text = mod_file
 
-                            # Expand shell variables and remediation functions into
-                            # corresponding XCCDF <sub> elements
+                            # Expand shell variables and remediation functions
+                            # into corresponding XCCDF <sub> elements
                             expand_xccdf_subs(fix, remediation_type, remediation_functions)
                     else:
                         sys.stderr.write("Skipping '%s' remediation script. "
-                                         "The platform identifier in the script is "
-                                         "missing!\n" % (filename))
+                                         "The platform identifier in the "
+                                         "script is missing!\n" % (filename))
         except OSError as e:
             if e.errno != errno.ENOENT:
                 raise
             else:
-                print("Not merging remediation scripts from the "
-                      "'%s' directory as the directory does not "
-                      "exist" % (fixdir))
+                sys.stderr.write("Not merging remediation scripts from the "
+                                 "'%s' directory as the directory does not "
+                                 "exist.\n" % (fixdir))
 
-    sys.stderr.write("Merged %d remediation scripts.\n"
-                     % (included_fixes_count))
+    sys.stderr.write("Merged %d %s remediations.\n"
```

Comment:
Actually I don't have a strong opinion about this. POSIX and GNU differ in convention AFAIK. Either way this PR doesn't change where we print this particular message so IMO raise it as a sepearate PR.